use-timescale/migration/hypershift-config.md

```diff
@@ -0,0 +1,322 @@
+---
+title: Configuring Hypershift
+excerpt: Migrate an existing PostgreSQL database to Timescale Cloud in a single step
+products: [cloud]
+keywords: [data migration, Hypershift]
+tags: [ingest, Hypershift, postgresql]
+---
+
+# Hypershift configuration file
+
+Hypershift uses a YAML configuration file to determine how to set up your new
+Timescale database. You can create your own file, or use the example file as a
+starting point. To complete your file, you need these details for the tables
+that you want to convert to hypertables:
+
+*   The schema that contains the table
+*   The name of the table
+*   The name of the `time` column of that table
+*   The chunk time interval to use
+*   The compression policy you want to use
+
+If you are not sure what chunk time interval to use, see the
+[time partitioning section][chunk-time].
+
+Use this format:
+
+```yml
+source: '<SOURCE_DB_URI>'
+target: '<TARGET_DU_URI>'
+actions: ['clone', 'verify']
+include_tables: []
+exclude_tables: []
+include_schemas: []
+exclude_schemas: []
+
+hypertable_configs:
+  name: <table_name>
+  schema: <schema_name>
+  time_column_name: <time_column_name>
+  chunk_time_interval: 7d
+  compress:
+    after: <in units of time>
+    segmentby:
+<column list>
+    orderby:
+<column list>
```

Comment:
We should also mention the direction field, as mentioned [here](https://github.com/timescale/hypershift/blob/1c11cf25001d69606bb79ea860b648c56f5cf8ae/src/config/mod.rs#L153)

```suggestion
- <column> <direction>
```